src/components/map/layers/TrackedEntityLayer.js

```diff
@@ -27,43 +34,43 @@ const getCentroid = (points) => {
 
 const fetchTEI = async (id, fieldsString, baseUrl) => {
     const data = await apiFetchWithBaseUrl({
-        url: `/trackedEntityInstances/${id}?fields=${fieldsString}`,
+        url: `/tracker/trackedEntities/${id}?fields=${fieldsString}`,
         baseUrl,
     })
     return data
 }
 
-const geomToCentroid = (type, coords) => {
-    switch (type) {
-        case 'POINT':
-            return JSON.parse(coords)
-        case 'POLYGON':
+const geomToCentroid = (geometry) => {
+    switch (geometry.type) {
+        case GEO_TYPE_POINT:
+            return geometry.coordinates
```

Comment:
Does it matter that this was switched from all caps `POINT` to `Point` ?

Reply:
Yes, now the tracker endpoint returns geometries in geojson format (ie type: `'Point'`, not all caps, and the coordinates are json object, not a string anymore). 
In some parts all caps types are still returned (eg. trackedEntityTypes).

Ref: 
- https://test.e2e.dhis2.org/analytics-2.41/api/tracker/trackedEntities/jZRaFaYkAtE
- https://test.e2e.dhis2.org/analytics-2.41/api/trackedEntityTypes/We9I19a3vO1